src/data/ab-attrs/post-faint-contact-damage-ab-attr.ts

```diff
@@ -1,4 +1,4 @@
-import { applyAbAttrs } from "#app/data/ability";
+import { applyAbAttrs } from "../apply-ab-attrs";
```

Comment:
```suggestion
import { applyAbAttrs } from "#app/data/apply-ab-attrs";
```
There are some more examples of this in other ab attrs; you could probably use a search and replace